apps/ledger-live-mobile/scripts/sync-families-dispatch.sh

```diff
@@ -4,18 +4,21 @@ set -e
 cd $(dirname $0)
 
 targets="\
-operationDetails \
-accountActions \
-TransactionConfirmFields \
-AccountHeader \
-AccountBodyHeader \
-AccountSubHeader \
-SendAmountFields \
-screens \
-SendRowsCustom \
-SendRowsFee \
-AccountBalanceSummaryFooter \
-SubAccountList \
+operationDetails.js \
+accountActions.js \
+TransactionConfirmFields.js \
+AccountHeader.js \
+AccountBodyHeader.js \
+AccountSubHeader.js \
+SendAmountFields.js \
+screens.js \
+SendRowsCustom.js \
+SendRowsFee.js \
+AccountBalanceSummaryFooter.js \
+SubAccountList.js \
+Confirmation.js \
+ConnectDevice.js \
+NoAssociatedAccounts.js
```

Comment:
@jackthta Can you please remind me what that change was for? Is it necessary?

Reply:
@haammar-ledger 

Oh yes, this change is for the custom components defined in `apps/ledger-live-mobile/src/families/hedera/*`. They are to be injected via `byFamily` in their respective screens:
- `apps/ledger-live-mobile/src/screens/AddAccounts/03-Accounts.js`
- `apps/ledger-live-mobile/src/screens/ReceiveFunds/02-ConnectDevice.js`
- `apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.js`